File: 260911/task 3/code.cs
namespace TestCS
{
    internal class Program
    {
        static void Main(string[] args)
        {
            float c;

            Console.Write("Temperature in Celsius << ");
            c = Convert.ToSingle(Console.ReadLine());

            Console.WriteLine($"\nFahrenheit >> {Math.Round((c * 9 / 5 + 32), 2)}" +
                $"\nKelvin >> {Math.Round((c + 273.15),2)}\n\nPress ENTER to exit...\n");

            Console.ReadLine();
        }
    }
}

File: 260911/task 3/code.py
c: float = float(input("Temperature in Celsius << "))

print(f"\nFahrenheit >> {round(c * 9 / 5 + 32)}" +
      f"\nKelvin >> {round(c + 273.15)}\n\nPress ENTER to exit...\n")

input()